plugins/module_utils/_api_data.py

```diff
@@ -844,6 +844,19 @@ def join_path(path):
             )),
         ],
     ),
+    ('routing', 'filter', 'num-list'): APIData(
+        versioned=[
+            ('7', '>=', VersionedAPIData(
+                fully_understood=True,
```

Comment:
Should this have a primary key? I think `("list", "range")` might make sense - or even `("list", )`? (I don't have access to a device right now so I cannot try this out.)

Reply:
Yeah, I was fiddling around with the `primary_keys=('list', 'range')` and `primary_keys=('list', )`, but I don't see any difference in the output.

Or at least, I doesn't change the order, all though I've `ensure_order: true`, `handle_absent_entries: 'remove'` and `handle_entries_content: 'remove'`.

So I'm a little bit clueless, at this moment.

Reply:
The question is whether you can have more than one entry with the same `list` or not. If not: `primary_keys=('list', )`. Otherwise, if you can have only one entry with the same `list` and `range`, then it's `primary_keys=('list', 'range')`. If not, then it's something else...

Reply:
It's the latter. You could have multiple `range` on the same `list`:

<img width="651" alt="image" src="https://github.com/user-attachments/assets/209b55b2-4c69-4aec-8f82-25043b060ffe">

Reply:
Cool! But you cannot have multiple entries with the same `list` and `range`, right?

Reply:
<img width="581" alt="image" src="https://github.com/user-attachments/assets/6c25061e-3d13-4127-b644-bb1bd56f170e">
Apparently, that's possible :)

So, no "uniques", and therefore no "primary" at all? Besides the `id`.

Reply:
Hmm, that's weird :) Yeah, in that case no primary key is probably best...